gui/tasks/forms.py

```diff
@@ -186,6 +187,8 @@ def check_rpath_exists(self):
         exists or not. Returns TRUE rpath is a directory
         and exists, else FALSE"""
 
+        self.ssh_host_keyscan()
```

Comment:
I tried adding it on save() as you suggested, but the fix doesn't work if I add it there.